platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/migratable/BundleProcessor.java

```diff
@@ -0,0 +1,355 @@
+   *     otherwise
+   */
+  public boolean installBundle(
+      BundleContext context, ProfileMigrationReport report, Bundle bundle) {
+    final String name = JsonBundle.getFullName(bundle);
+    final String attempt = report.getBundleAttemptString(Operation.INSTALL, name);
+
+    LOGGER.debug("Installing bundle '{}'{}", name, attempt);
+    report.record("Installing bundle [%s]%s.", name, attempt);
+    try {
+      context.installBundle(bundle.getLocation());
+    } catch (IllegalStateException | BundleException | SecurityException e) {
+      report.recordOnFinalAttempt(
+          new MigrationException("Import error: failed to install bundle [%s]; %s.", name, e));
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   * Uninstalls the specified bundle.
+   *
+   * @param report the report where to record errors if unable to uninstall the bundle
+   * @param bundle the bundle to uninstall
+   * @return <code>true</code> if the bundle was uninstalled successfully; <code>false</code>
+   *     otherwise
+   */
+  public boolean uninstallBundle(ProfileMigrationReport report, Bundle bundle) {
+    final String name = JsonBundle.getFullName(bundle);
+    final String attempt = report.getBundleAttemptString(Operation.UNINSTALL, name);
+
+    LOGGER.debug("Uninstalling bundle '{}'{}", name, attempt);
+    report.record("Uninstalling bundle [%s]%s.", name, attempt);
+    try {
+      bundle.uninstall();
+    } catch (IllegalStateException | BundleException | SecurityException e) {
+      report.recordOnFinalAttempt(
+          new MigrationException("Import error: failed to uninstall bundle [%s]; %s.", name, e));
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   * Starts the specified bundle.
+   *
+   * @param report the report where to record errors if unable to start the bundle
+   * @param bundle the bundle to start
+   * @return <code>true</code> if the bundle was started successfully; <code>false</code> otherwise
+   */
+  public boolean startBundle(ProfileMigrationReport report, Bundle bundle) {
+    final String name = JsonBundle.getFullName(bundle);
+    final String attempt = report.getBundleAttemptString(Operation.START, name);
+
+    LOGGER.debug("Starting bundle '{}'{}", name, attempt);
+    report.record("Starting bundle [%s]%s.", name, attempt);
+    try {
+      bundle.start();
+    } catch (IllegalStateException | BundleException | SecurityException e) {
+      report.recordOnFinalAttempt(
+          new MigrationException("Import error: failed to start bundle [%s]; %s.", name, e));
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   * Stops the specified bundle.
+   *
+   * @param report the report where to record errors if unable to stop the bundle
+   * @param bundle the bundle to stop
+   * @return <code>true</code> if the bundle was stopped successfully; <code>false</code> otherwise
+   */
+  public boolean stopBundle(ProfileMigrationReport report, Bundle bundle) {
```

Comment:
👍  Fair enough. 

We are only saving on logs and exception handling in favor of an additional method call and object creating for the lambda that will be passed in for actual execution. I also changed the feature processor but not the application processor for two reasons. The first, being that it only has one operation and the second being that it will go away in the next PR.